akka-actor-typed-tests/src/test/java/jdocs/akka/typed/StyleGuideDocExamples.java

```diff
@@ -672,14 +674,17 @@ public Value(int value) {
         }
       }
 
-      // Tick is a PrivateCommand so can't be sent to an ActorRef<Command>
-      enum Tick implements PrivateCommand {
+      // The type of the Counter actor's internal event messages.
+      public interface Event extends Message {}
```

Comment:
this should still be private? No code outside of this scope should create / extend it

Reply:
Thanks, @chbatey, together the changes work.

Just doing `Event` was giving me a

```
private trait Event escapes its defining scope as part of type jdocs.akka.typed.StyleGuideDocExamples.PublicVsPrivateMessages2.Counter.Event
```

error - and I guess I was having a slow day.